packages/email-service/jest.config.ts

```diff
@@ -12,4 +12,7 @@ module.exports = {
   },
   moduleFileExtensions: ['ts', 'js', 'html'],
   coverageDirectory: '../../coverage/packages/email-service',
+  collectCoverage: true,
```

Comment:
I disagree this is should be set by default. I can use --coverage when I want to see a coverage.